package.json

```diff
@@ -3,75 +3,71 @@
   "private": true,
   "appName": "BCRS Shared Components",
   "scripts": {
-    "storybook": "start-storybook -p 6006",
-    "storybook:build": "build-storybook -o ./docs",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
+    "storybook": "storybook dev",
+    "build-storybook": "storybook build -o ./docs",
+    "lint": "eslint --ext .js,.ts,.vue .",
     "test:unit": "vue-cli-service test:unit --testPathPattern",
     "test:unit-silent": "vue-cli-service test:unit --silent",
     "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
```

Comment:
I need to fix these 3 which still reference vue-cli. I'll have to research what the Vite replacement is.

Reply:
Fixed and working!